src/main/java/org/folio/search/cql/LccnSearchTermProcessor.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.search.cql;
+
+import lombok.RequiredArgsConstructor;
+import org.folio.search.service.setter.instance.LccnInstanceProcessor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class LccnSearchTermProcessor implements SearchTermProcessor {
+
+  private final LccnInstanceProcessor lccnProcessor;
+
+  @Override
+  public String getSearchTerm(String inputTerm) {
+    return lccnProcessor.normalizeLccn(inputTerm);
```

Comment:
This search term processor could be used in future for authorities searching. So it's not good to have smth related to instance in it. I suggest to move normalizeLccn method to some utility class, then it will not be needed to inject instance processor into this class.